src/cardboard-distorter.js

```diff
@@ -98,8 +98,7 @@ function createRenderTarget(renderer) {
   var parameters = {minFilter: THREE.LinearFilter,
                     magFilter: THREE.LinearFilter,
                     format: THREE.RGBFormat,
-                    stencilBuffer: false,
-                    depthBuffer: false};
```

Comment:
Yes. As far as I understand, cardboard-distorter basically distorts a picture, there is no depthBuffer usage.